scripts/data/convert_general_thought_to_tulu_thinker.py

```diff
@@ -0,0 +1,27 @@
+from datasets import load_dataset, Dataset
```

Comment:
Maybe a short doc on what the script is and commands to run them?

https://github.com/allenai/open-instruct/blob/b5f60321d8186500d10660e3d7b575ae619a678b/scripts/data/rlvr/gsm8k_rlvr.py#L1-L10